tvtk/tests/test_tvtk.py

```diff
@@ -200,7 +200,7 @@ def test_object_cache(self):
         # The test sometimes fails as VTK seems to generate objects with the
         # same memory address and hash, we try to force it to allocate more
         # objects so as to not end up reusing the same address and hash.
-        junk = [vtk.vtkConeSource() for i in range(5)]
+        junk = [tvtk.ConeSource() for i in range(50)]
```

Comment:
One of those unreliable test which started failing. Changing to `tvtk` object and allocating more objects (hence increased the range) results in different address and hash.